client/my-sites/themes/current-theme/index.tsx

```diff
@@ -12,31 +11,40 @@ import InlineSupportLink from 'calypso/components/inline-support-link';
 import withBlockEditorSettings from 'calypso/data/block-editor/with-block-editor-settings';
 import { isFullSiteEditingTheme } from 'calypso/my-sites/themes/is-full-site-editing-theme';
 import { getActiveTheme, getCanonicalTheme } from 'calypso/state/themes/selectors';
+import { Theme } from 'calypso/types';
 import { trackClick } from '../helpers';
 import { connectOptions } from '../theme-options';
 
 import './style.scss';
 
+interface CurrentThemeProps {
+	blockEditorSettings: {
+		is_fse_eligible: boolean;
+	};
```

Comment:
We don't need to redefine that. We can:

```suggestion
	blockEditorSettings: BlockEditorSettings;
```

You'll need to import `BlockEditorSettings` that is already exported from `calypso/data/block-editor/use-block-editor-settings-query`

Reply:
Good catch.👍